package.json

```diff
@@ -23,6 +23,10 @@
             { 
                 "command": "extension.insertChild",
                 "title": "Org-mode: Insert Child"
+            },
+            {
+                "command": "extension.insertDate",
+                "title": "Org-mode: Insert Date"
```

Comment:
Mind using the title from `feature_queue.org`: "Org: Time Stamp"? The official org name is `org-time-stamp`. I'm down for either capitals with spaces or kebab case.